cmd/cmd_pkg.go

```diff
@@ -230,6 +229,24 @@ func loadCommandHandler(cmd *cobra.Command, args []string) {
 		exitWithError(err.Error())
 	}
 
+	// Validate local package
+	packageFolder := filepath.Base(pkgFlags.PackagePath())
+	executableName := c.Program
+	if strings.Contains(executableName, packageFolder) {
+		executableName = filepath.Base(executableName)
+	}
+	executablePath := filepath.Join(pkgFlags.PackagePath(), executableName)
+
+	if runtime.GOOS == "linux" {
```

Comment:
Does it make sense to do this for `darwin`?

Reply:
yeh - no need to ensure a platform chk here - the real problem is when someone is building on a mac and forgets to target for linux